src/api/java/com/minecolonies/api/colony/requestsystem/requestable/Food.java

```diff
@@ -169,7 +171,7 @@ public static Food deserialize(final IFactoryController controller, final Packet
     @Override
     public boolean matches(@NotNull final ItemStack stack)
     {
-        return stack.getItem().isFood() && !exclusionList.contains(new ItemStorage(stack));
+        return stack.getItem().isFood() && !exclusionList.contains(new ItemStorage(stack)) && !(ItemStackUtils.ISCOOKABLE.test(stack) && exclusionList.contains(new ItemStorage(MinecoloniesAPIProxy.getInstance().getFurnaceRecipes().getSmeltingResult(stack))));
     }
```

Comment:
We have to consider there are two things
a) There are things that will be delivered from the warehouse for the cook to hand out directly
b) There is food that we want the cook to request to cook then.

Does this handle both cases?

Reply:
Yes